include/exadg/compressible_navier_stokes/postprocessor/postprocessor.h

```diff
@@ -64,11 +64,13 @@ class PostProcessor : public PostProcessorBase<dim, Number>
 
   virtual ~PostProcessor();
 
-  virtual void
-  setup(Operator<dim, Number> const & pde_operator);
+  void
+  setup(Operator<dim, Number> const & pde_operator) override;
```

Comment:
`virtual` actually makes sense for postprocessors

Reply:
I would be fine either way, so we need to decide about the style how we declare overriden virtual functions - do it as @bergbauer suggests with simply `override` and no `virtual`, or `virtual void ... override` in the verbose version.

Reply:
I am also fine with both options but I favor the short version. What is the style in dealii?

Reply:
I checked in deal.II and it seems that it is mostly `virtual ... override`, e.g. here:
https://github.com/dealii/dealii/blob/981214b60b491f465dd8cf04e24989a2d806229c/include/deal.II/multigrid/mg_matrix.h#L78-L85
or here:
https://github.com/dealii/dealii/blob/981214b60b491f465dd8cf04e24989a2d806229c/include/deal.II/dofs/dof_handler_policy.h#L132-L133

Reply:
the `virtual ... override` combination is what I thought of originally